opus/application/static_media/js/opus.js

```diff
@@ -336,11 +335,11 @@ var opus = {
         opus.widgets_drawn = [];
         opus.widget_elements_drawn = [];
 
-        if (resetMetadata && !opus.checkIfMetadataAreDefault()) {
+        if (resetMetadata && !o_utils.areObjectsEqual(opus.prefs.cols, default_columns.split(','))) {
```

Comment:
If I'm reading this code right, it seems cleaner to take the areObjectsEqual out first:
```
if (!areObjectsEqual...) {
   if (resetMetadata) {
      do thing 1
   } else {
      do thing 2
   }
}